addons/web_editor/static/src/js/editor/odoo-editor/src/OdooEditor.js

```diff
@@ -4417,7 +4417,6 @@ export class OdooEditor extends EventTarget {
 
             if (isMouseInsideCheckboxBox) {
                 toggleClass(node, 'o_checked');
-                ev.preventDefault();
```

Comment:
Could you update the commit message to explain why the `preventDefault` was creating the issue in the first place? What other piece of code somewhere else was prevented to run because of it?

Reply:
@dmo-odoo I have updated the commit message!

Reply:
Thanks! 👍